pyproject.toml

```diff
@@ -9,6 +9,5 @@ select = ["F"]
 ignore = [
   "F401", # unused imports
   "F811", # Redefinition of unused variable
-  "F821", # Undefined name
```

Comment:
The main point here is that removing F821 from the ignore list,
and making sure that pre-commit CI passes.